dbus-codegen/src/generate.rs

```diff
@@ -513,7 +513,7 @@ fn write_server_access(s: &mut String, i: &Intf, saccess: ServerAccess, minfo_is
             *s += "        let d = dd.as_ref();\n";
         },
         ServerAccess::RefClosure => *s += &format!("        let d = fclone({}minfo);\n", z),
-        ServerAccess::MethodInfo => *s += &format!("        let d: &{} = {}minfo;\n", make_camel(&i.shortname), z),
+        ServerAccess::MethodInfo => *s += &format!("        let d: &dyn {} = {}minfo;\n", make_camel(&i.shortname), z),
     }
```

Comment:
Hmm, I guess the dynamic dispatch can be avoided but it's complicated so I think this is okay for now.

Reply:
I just wanted to fix the lint about [bare trait objects](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#bare-trait-object). Didn't know that it has implications other than to explicit the use of traits.